File: backend/domain/src/main/kotlin/pt/ira/Evidence.kt
package pt.ira

data class Evidence (
    val id: Int,
    val type: String,      // JSONB
    val filePath: String,
    val location: String,
    val description: String,
    val reporterId: Int?,
    val reportId: Int?,
    val createdAt: Long = System.currentTimeMillis(),
    val updatedAt: Long = System.currentTimeMillis(),
)

File: frontend/movel/models/evidence/Evidence.ts
import type { Json } from "../utils/Json";

export interface Evidence {
    id: number;
    type: Json;
    filePath: string;
    location: string;
    description: string;
    reporterId: number;
    reportId: number;
    createdAt: number;
    updatedAt: number;
}

